broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -96,11 +100,29 @@ async function commit ({ params, relayer, logger, engines, orderbooks }, { Empty
     throw new PublicError(errorMessage)
   }
 
-  await engine.createChannel(address, balance)
+  logger.debug('Creating outbound channel')
+
+  try {
+    await engine.createChannel(address, balance)
+  } catch (e) {
+    logger.error('Received error when creating outbound channel', { e })
+
+    if (e.details && e.details.includes(CHANNEL_FUNDING_ERROR)) {
+      throw new PublicError(`Failed to commit wallet: Your wallet balance cannot cover on-chain fees. Please commit a smaller balance`)
```

Comment:
is this the only way this error gets triggered? Being user-friendly is nice, but if there is a case that triggers this error that isn't the one we give to the user it will be very confusing.

Reply:
I dont like this either, because that error (string matching) can potentially change, but for now _i think_ this is OK. We have work coming up that will deal with fees that will make this error obsolete (for the most part).